devcontainer.docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+---
+version: '3.8'
```

Comment:
The 3.8 compose version was added in docker-compose 1.25.5, ubuntu has only 1.25.0 as stable in his 20.04 repositories. 
I had to change this to 3.7 and everything went well.

Reply:
thanks, will change it 3.7